src/store/wallet.js

```diff
@@ -63,9 +63,11 @@ export default {
     setVTT (state, transaction) {
       state.vtt = transaction
     },
-
-    setGeneratedAddress (state, { address }) {
-      state.generatedAddress = address
+    setAddresses (state, { addresses }) {
```

Comment:
It would be nice to have only one method that includes both. What do you think about changing it?